changes/pr2774.yml

```diff
@@ -0,0 +1,2 @@
+task:
+  - 'Add link to Prefect Server Survey to README and Docs - [#2774](https://github.com/PrefectHQ/prefect/pull/2774)'
```

Comment:
This isn't really a "task" related change (task meaning the task library). I think this doesn't need a changelog entry at all actually.

Reply:
Makes sense, I didn't fully read the change example I was kind of rushing 😅